src/main/java/eggventory/enums/Property.java

```diff
@@ -0,0 +1,5 @@
+package eggventory.enums;
+
```

Comment:
have to deconflict this with the StockProperty enum that raghav changed the name to in #249